### PR TITLE
added files with code

### PR DESCRIPTION
**Pull request by Szpytowski**
Added a new branch and created a pull request just as asked.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+<div class="resume-container">
+    <header>
+        <h1>CHARLES MCTURLAND</h1>
+        <p class="title">SOFTWARE ENGINEER</p>
+    </header>
+    <div class="content">
```
Comment (Solnick) [inline index.html:18]: I think you can use a semantic tag here rather than a div

File: index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+<div class="resume-container">
```
Comment (Solnick) [inline index.html:13]: this div is a child of body, so it should be indented a little bit

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    color: #0073e6;
+    text-decoration: none;
+}
+
+.main-content {
+    width: 70%;
+    padding-left: 10px;
+}
+
+.linkedin-container{
```
Comment (Solnick) [inline css/style.css:86]: remember about a space after the class name

**Review by Solnick — COMMENTED** [review]
image
Remember to add margins at the right side of the text. Same goes for the occupation name at the top of the page

File: index.html (hunk 1 above)
Comment (Szpytowski) [reply index.html:18]: div

I tried to use something else, but it ruins the whole lay out. I also used flex on that div to keep everything inside intact.

Comment (Solnick) [reply index.html:18]: I don't think so. I've just tried to replace this `div` with the `main` tag and it works like a charm

**Pull request merged**